frontend/packages/dev-console/src/components/topology/shapes/BaseNode.scss

```diff
@@ -10,7 +10,7 @@
   }
 
   &__label {
-    pointer-events: none;
+    cursor: default;
```

Comment:
removing ```pointer-events: none;``` makes nodes name clickable and it's open the overview page.  I am not sure it is expected behavior. cc: @serenamarie125

Reply:
I kind of liked that I could open the details from the name thing.

Reply:
@vikram-raj got your point but if we'll have `pointer-events: none;` then how will address above issue of showing tooltip for resource badges [0] as with above even hover won't get triggered. 

Will file a new PR as this is merged once decided.

[0] https://jira.coreos.com/browse/ODC-1539

Reply:
@invincibleJai Currently, everything in node wrap under a single ```<g>``` tag and ```onClick``` event to open the overview added to it. If we can move label from that ```<g>``` tag put it in another ```<g>``` tag then we can resolve this issue.